multidc/src/main/scala/com/lightbend/re/ReplicatedEntityApp.scala

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2017 Lightbend Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.lightbend.re
+
+import java.io.File
+
+import akka.actor.{ActorRef, ActorSystem}
+import akka.cluster.Cluster
+import akka.cluster.http.management.ClusterHttpManagement
+import akka.cluster.sharding.{ClusterSharding, ClusterShardingSettings}
+import akka.persistence.multidc.PersistenceMultiDcSettings
+import com.lightbend.re.ReplicatedCounter.{Increment, ShardingEnvelope}
+import com.typesafe.config.ConfigFactory
+
+import scala.io.StdIn
+
+object ReplicatedEntityApp extends App {
+  val rootConfFile = new File("/home/akka/multidc/application.conf")
+  val rootConf =
+    if (rootConfFile.exists) ConfigFactory.parseFile(rootConfFile)
+    else ConfigFactory.empty("no-root-application-conf-found")
+  val conf = rootConf.withFallback(ConfigFactory.load())
+  println(s"Cloud configuration: ${rootConfFile.exists}")
+
+  implicit val system = ActorSystem("MultiDcSystem", conf)
+
+  val cluster = Cluster(system)
+  ClusterHttpManagement(cluster).start()
+
+  ClusterSharding(system).start(
+    typeName = ReplicatedCounter.ShardingTypeName,
+    entityProps = ReplicatedCounter.shardingProps(PersistenceMultiDcSettings(system)),
+    settings = ClusterShardingSettings(system),
+    extractEntityId = ReplicatedCounter.extractEntityId,
+    extractShardId = ReplicatedCounter.extractShardId)
+
+  val counterProxy: ActorRef = ClusterSharding(system).startProxy(
+    typeName = ReplicatedCounter.ShardingTypeName,
+    role = None,
+    dataCenter = Some("eu-west"),
+    extractEntityId = ReplicatedCounter.extractEntityId,
+    extractShardId = ReplicatedCounter.extractShardId)
+
+  HttpApi.startServer("localhost", 8080, counterProxy)
```

Comment:
The Sharding proxy can be used to send message to another DC, but is that what we want here?
Shouldn't it send to the entity in the same DC?

Note that `ClusterSharding(system).start` returns an `ActorRef` to the local sharding region, which should be used to send messages to entity in same DC.